### PR TITLE
SW/SKR Mini E3 v3 support

### PR DESCRIPTION
**Pull request by jharmn**
Pins remapped from v2->v3
Pin mappings tested on my SW

### CODE REVIEW

**Comment by jotpowers** [comment]
Can confirm that the new heater sensor pin is PC4 from my switchwire, so happy to see this config being submitted.

File: Firmware/skr_mini_e3_v3_config.cfg [hunk 1]
```diff
@@ -0,0 +1,403 @@
...
+max_power: 1.00
+kick_start_time: 0.200
+heater: heater_bed
+
+######
+# Hot End Fan
+# FAN2 Connector
+###############
+[heater_fan extruder_fan]
+pin: PC15
```
Comment (raymondh2) [inline Firmware/skr_mini_e3_v3_config.cfg:234]: This pin is incorrect

Comment (jharmn) [reply Firmware/skr_mini_e3_v3_config.cfg]: Good catch, corrected [9a7c9eb](https://github.com/VoronDesign/Voron-Switchwire/pull/44/commits/9a7c9ebe65667b751fb2e27f25636313386a9d0c)

**Comment by jharmn** [comment]
@luki343 you said "a few wrong pins", but only specified the PC15->PB15 (which has been corrected. Did you find any other incorrect mappings?

**Pull request merged**